IBM/carbon-addons-iot-react
# [GaugeCard] Increase font size and gauge stroke

Closes #1117 closes #1113 

**Summary**

Increased font size of gauge value and increase the stroke weight of the gauge. Also will remove the grade value until designs account for placement. 

**Change List (commits, features, bugs, etc)**

- tweak values and add classes in GaugeCard.jsx
- Change GaugeCard.story.jsx to use Cardminwidth utility 
- tweaked styles based off amount of digits in the value in _gauge-card.scss

**Acceptance Test (how to verify the PR)**

- go to the gauge card story
- use knobs panel to change value of gauge. 
- see sizes change with 2, 3, 4 digits
